sites/friday/src/get-started/django/deploy/configure.md

```diff
@@ -0,0 +1,107 @@
+
+1. **(Optional) Configure root**. The configuration generated assumes that the Django project root is the repository root.
+If this is not the case, and your Django code is nested within a directory, add the following `source.root` configuration to `{{% vendor/configfile "app" %}}`.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            source:
+                root: NESTED_DIR
+
+            # ...GENERATED CONFIGURATION...
+    ```
+
+1. **Configure the build and deploy**. The `{{% vendor/configfile "app" %}}` contains a `hooks` block, containing configuration for how Django should be built and deployed. 
+The configuration for the "From scratch" app is shown below.
+Update your own build and deploy steps using it as an example.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            hooks:
+                build: |
+                    set -eux
+
+                    # Upgrade pip
+                    python -m pip install --upgrade pip
+                    
+                    # Install dependencies
+                    pip install -r requirements/local.txt
+                    pip install -r requirements/production.txt
+
+                    # Collect static assets
+                    python manage.py collectstatic
+
+                deploy: |
+                    set -eux
+
+                    # Migrations.
+                    python manage.py migrate
+                    
+            # ...GENERATED CONFIGURATION...
+    ```
+
+1. **Configure how the app is started and served**. There is a `web` block in `{{% vendor/configfile "app" %}}` that describes a `start` command for Django, as well as other details for important path `locations` that should be served in specific ways. 
+The configuration for the "From scratch" app is shown below.
+Update your own build and deploy steps using it as an example.
+
+    ```yaml {configFile="app"}
+    applications:
+        APP_NAME:
+            # ...GENERATED CONFIGURATION...
+
+            web:
+                commands:
+                    start: "gunicorn -w 4 -b unix:$SOCKET APP_NAME.wsgi:application"
+
+                upstream:
+                    socket_family: unix
+
+                locations:
+                    "/":
+                        passthru: true
+                    "/static":
+                        root: "static"
+                        expires: 1h
+                        allow: true
+
+            # ...GENERATED CONFIGURATION...
+    ```
+
+    The example above uses a Gunicorn server to run the application.
+    Be sure to substitute the placeholder `APP_NAME` with your own application name.
+    For the "From scratch" skeleton project, replace `APP_NAME` with `my_awesome_project`.
+
+1. Update `.environment`?
+
+{{% /guides/deploy/configure/project-init %}}
```

Comment:
because we're including this shortcode (?) the numbering gets confusing. We go from step 4 and then back to step 1. 

![image](https://github.com/platformsh/platformsh-docs/assets/2332956/903217df-5f18-4f7a-9c2d-709b65be195f)